### DISCUSSION TITLE
Pin top / relevant comments

### DISCUSSION THREAD

**Discussion by luisquintanilla** [community/community#18523] [Projects and Issues] [answered]
I'd like the ability to pin a comment at the top of an issue thread indicating it's either the resolution or more relevant response thus far so that I don't have to scroll through the entire thread to know what the plan of action is.

**Comment by zadjii-msft** [comment]
Just to x-ref:
* #47912
* #27522
* #18523

all seem like they're the same request.

**Reply by gnowland** [reply]
Also https://github.com/isaacs/github/issues/1008 (which contains some examples and references to related feature requests) and https://github.com/isaacs/github/issues/1555 from the now archived `isaacs/github` repository.

**Comment by azenMatt** [comment, marked as answer]
This is now possible! https://github.blog/changelog/2026-02-05-pinned-comments-on-github-issues/
